root/url/UrlHeader.js

```diff
@@ -18,7 +18,7 @@ type Props = {|
   url: UrlT,
 |};
 
-const URLHeader = ({url, page}: Props) => (
+const UrlHeader = ({url, page}: Props) => (
```

Comment:
Should ideally be renamed in a separate commit. `root/url/layout.tt` be updated too.

Reply:
Changed